src/ServicePulse.Host/vue/src/components/monitoring/D3LargeGraph.stories.js

```diff
@@ -0,0 +1,57 @@
+import D3LargeGraph from "./D3LargeGraph.vue";
+/*import { expect } from "@storybook/jest";*/
+export default {
+    component: D3LargeGraph,
+    title: "Monitoring/EndpointDetails/D3LargeGraph",
+    tags: ["autodocs"],
+    //👇 Our events will be mapped in Storybook UI
+    argTypes: {},
+};
+
+//stories
+
+export const Default = {
+
+    minimumyaxis: 10,
+    avglabelcolor: "#EA7E00",
+    isdurationgraph: false,
+    metricsuffix: "MSGS",
+    csclass: "large-graph pull-left",
```

Comment:
I think we can encapsulate these classes and also remove the following properties: firstseriescolor: "#EA7E00",
    firstseriesfillcolor: "#EADDCE",
    secondseriescolor: "",
    secondseriesfillcolor: "",

Instead, let the graph component use CSS specificity to show the corresponding colors. For example, see how the static version uses CSS specificity:

<img width="990" alt="image" src="https://github.com/Particular/ServicePulse/assets/545695/c3bdd49b-981e-4564-a7f0-39c6097f6a83">

All the different colors are already in the CSS in a way where it is expected only to set a css marker class, which establishes the specificity 

<img width="542" alt="image" src="https://github.com/Particular/ServicePulse/assets/545695/9045f14e-3bb5-400f-a883-26372058b652">

Reply:
I agree that there are many redundancies that needs to be removed and code optimized. I was planning to do that once I got all the pieces of the graph component working .

Reply:
Sounds good @soujay I will approve the PR based on that plan.